app/scripts/dcmWalkingRetargeting.xml

```diff
@@ -58,6 +58,8 @@
     <dependencies>
       <port timeout="5.0">/transformServer/transforms:o</port>
       <port timeout="5.0">/joypadDevice/Oculus/rpc:i</port>
+      <port timeout="5.0">/icub/cam/left</port>
```

Comment:
I added these lines since the resetting of the cameras occurs as soon as we start the module. If ``resetCameras`` is enabled, it needs to connect to these ports immediately.